launchflow/flows/unlock_flows.py

```diff
@@ -0,0 +1,55 @@
+from launchflow.locks import LockOperation, OperationType
+from launchflow.managers.resource_manager import ResourceManager
+from launchflow.managers.service_manager import ServiceManager
+from launchflow.models.enums import ResourceStatus, ServiceStatus
+
+
+async def unlock_resource(resource_manager: ResourceManager):
+    # First force unlock the resource
+    resource = await resource_manager.load_resource()
+    try:
+        await resource_manager.force_unlock_resource()
+    except Exception as e:
+        if not resource.status.is_pending():
+            raise e
```

Comment:
why raise if not pending?

Reply:
cause if it's not pending it means it was not locked and we don't need to update the status